hack/image_patch_dev.sh

```diff
@@ -77,4 +77,9 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "100m"
     }
+    metricsAggregator: |-
```

Comment:
@alexagriffith @yuzisun : This `metricsAggregator` is improperly intended and hence it is getting inserted as part of router config and causing json marshalling errors in inference graph controller. 
Fixing it in https://github.com/kserve/kserve/pull/2539